ui/packages/app/src/api_docs/ApiDocs.svelte

```diff
@@ -0,0 +1,176 @@
+<script lang="ts">
```

Comment:
I think there is more whitespace now between the bottom of the demo and the `View API` button? This is with the model3D demo

### This PR
![view_api_whitespace_pr](https://user-images.githubusercontent.com/41651716/210424553-45cfa9e5-de17-4cc4-b1af-832ef1374273.gif)

### Main
![view_api_whitespace_main](https://user-images.githubusercontent.com/41651716/210424582-c521ddff-3887-475a-9075-8298b262910b.gif)